src/command.c

```diff
@@ -702,13 +711,16 @@ static int dv_convert_fc_v0(struct device *dev, struct dmp_cmb *cmb,
 			quant_base = dma_buf_kmap(dma_buf, ++page_num);
 			if (!quant_base) {
 				pr_err(DRM_DEV_NAME ": dma_buf_kmap() failed\n");
-				dma_buf_end_cpu_access(dma_buf, DMA_BIDIRECTIONAL);
+				dma_buf_end_cpu_access(dma_buf,
+							DMA_BIDIRECTIONAL);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket